.ci/setup_env_opensuse.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+echo "Install chronic"
+sudo -E zypper -n install moreutils
+
+echo "Install curl"
+sudo -E zypper -n install curl
```

Comment:
as we install `chronic` first, should we use it on these next two items - we seem to use it on all the ones after that... I wonder if it is just a copy/paste/oversight?